pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.github.mivek</groupId>
     <artifactId>metarParser</artifactId>
-    <version>1.10.3</version>
+    <version>1.10.4</version>

```

Comment:
Sorry but the CONTRIBUTING.md is not up to date. Can you revert this, the CI will update the version.

I have a PR to clean the CONTRIBUTING.md